omod/src/test/java/org/openmrs/module/idgen/web/controller/IdentifierSourceControllerTest.java

```diff
@@ -42,50 +42,49 @@
  * Tests methods in IdentifierSourceController
  */
 public class IdentifierSourceControllerTest {
-
+	
 	@Autowired
 	@InjectMocks
 	private IdentifierSourceController controller;
+	
 	@Mock
 	private IdentifierSourceService iss;
-
+	
 	@Before
 	public void setUp() {
 		MockitoAnnotations.initMocks(this);
 	}
 	
 	@Test
 	public void exportIdentifiers_shouldReturnJson() throws Exception {
-		Mockito.stub(iss.generateIdentifiers(
-				Mockito.any(IdentifierSource.class), 
-				Mockito.any(Integer.class), 
-				Mockito.any(String.class))).toReturn(Arrays.asList("1", "2", "3"));
-		
+		Mockito.stub(iss.generateIdentifiers(Mockito.any(IdentifierSource.class), Mockito.any(Integer.class),
+		    Mockito.any(String.class))).toReturn(Arrays.asList("1", "2", "3"));
```

Comment:
Can you explain the reason behind changing this class?

Reply:
I opened this class , and after doing a ctr-shift-s  it was automatically formated using the openMRS formatter that I intergrated for eclipse.

Reply:
Can you discard all the changes in this file?